src/system_tray.cpp

```diff
@@ -10,10 +10,17 @@
     #include <accctrl.h>
     #include <aclapi.h>
     #define TRAY_ICON WEB_DIR "images/favicon.ico"
-  #elif defined(__linux__) || defined(linux) || defined(__linux)
+    #define TRAY_ICON_PLAYING WEB_DIR "images/sunshine-playing.ico"
+    #define TRAY_ICON_PAUSING WEB_DIR "images/sunshine-pausing.ico"
+    #define TRAY_ICON_LOCKED WEB_DIR "images/sunshine-locked.ico"
+  #else
     #define TRAY_ICON "sunshine"
-  #elif defined(__APPLE__) || defined(__MACH__)
-    #define TRAY_ICON WEB_DIR "images/logo-sunshine-16.png"
+    #define TRAY_ICON_PLAYING "sunshine-playing"
+    #define TRAY_ICON_PAUSING "sunshine-pausing"
+    #define TRAY_ICON_LOCKED "sunshine-locked"
+  #endif
```

Comment:
Should we leave the conditions for linux and macos? From my earlier testing the tray icon required png on macos.

Or maybe just a comment about it?

Reply:
I can change it, but in theory the macOS tray icon does not work, am I right?

Reply:
It does not work with Sunshine, but the tray library does work with macOS.

I couldn't figure out how to make it not block the main thread of sunshine.